PySoar/analysis_run.py

```diff
@@ -0,0 +1,20 @@
+from competitionDay import CompetitionDay
```

Comment:
Oh sorry, I tried something and it accidentally made it into a commit. Line can be deleted